### DISCUSSION TITLE
No command handler found, still WIP

### DISCUSSION THREAD

**Discussion by mikefairbrother-rldatix** [discordjs/discord.js#10133] [Q&A]
I really don't understand what I am missing here and it's frustrating.
I've started a really simple bot. It loads, it responds to a simple hello message and a /ping command works perfectly.

I've added a new command called coinflip.

It resides in /commands/fun/ and is coinflip.js
It is registered and I can see it as a slash command to use.

Every time I run it, the response is "No command handler found, still WIP" 

```
const { SlashCommandBuilder, ChatInputCommandInteraction, EmbedBuilder } = require('discord.js');

module.exports = {
    data: new SlashCommandBuilder()
        .setName('coinflip')
        .setDescription('Flip a coin to get Heads or Tails from the bot')
        .addStringOption(option =>
            option.setName('choice')
                .setDescription('Choose Heads or Tails (optional)')
                .addChoices(
                    { name: 'Heads', value: 'Heads' },
                    { name: 'Tails', value: 'Tails' }
                )),

async execute(interaction) {
    if (!(interaction instanceof ChatInputCommandInteraction)) return;

    const choice = interaction.options.getString('choice');
    const outcomes = ['Heads', 'Tails'];
    const result = outcomes[Math.floor(Math.random() * outcomes.length)];

    const response = choice ? `Your choice: **${choice}**\Flip Result: **${result}**` : `Flip Result: **${result}**`;

    if (interaction.deferred) {
        await interaction.editReply(response);
    } else {
        await interaction.reply(response);
    }
},
};
```

I know this will be something stupid, but I really don't understand what I am missing here. The ping.js is in command/utility and this new one is in command/fun. The commands are loaded dynamically etc, I've basically followed the guide to this point and now I try and add a custom command, it's falling over.

**Comment by sdanialraza** [comment]
The issue is to do with your command handler or your interactionCreate event, you should join the Discord server and ask there and we'd be happy to help
